Objects/regions.c

```diff
@@ -1157,10 +1176,29 @@ static PyObject *PyRegion_open(PyRegionObject *self, PyObject *args) {
     Py_RETURN_NONE;  // Return None (standard for methods with no return value)
 }
 
+int _PyRegion_is_closed(PyObject* self) {
+    return PyRegion_is_open((PyRegionObject *)self, NULL) == Py_False;
+}
+
+
 // Close method (sets the region to "closed")
+// TODO: integrate with #19 and associated PRs
 static PyObject *PyRegion_close(PyRegionObject *self, PyObject *args) {
-    regionmetadata_close(self->metadata);  // Mark as closed
-    Py_RETURN_NONE;  // Return None (standard for methods with no return value)
+    regionmetadata * const md = Py_REGION(self);
```

Comment:
GH CI is complaining about the use of `Py_REGION`. In `regions.c` you can use `Py_REGION_DATA` to directly cast to `regionmetadata *`